src/main/resources/com/google/api/codegen/go/main.snip

```diff
@@ -116,7 +116,7 @@
     // use by Google-written clients.
     func (c *{@view.clientTypeName}) SetGoogleClientInfo(name, version string) {
         goVersion := strings.Replace(runtime.Version(), " ", "_", -1)
-        v := fmt.Sprintf("%s/%s %s gax/%s go/%s", name, version, gapicNameVersion, gax.Version, goVersion)
+        v := fmt.Sprintf("%s/%s %s gax/%s gl-go/%s", name, version, gapicNameVersion, gax.Version, goVersion)
```

Comment:
The constructor is actually setting the following:

```
    c.SetGoogleClientInfo("gax", gax.Version)
```

IIUC, the Veneers don't call `NewClient()`? Or do they, but then immediately follow it with a `SetGoogleClientInfo()` call?

If the latter, do we know that there's only one `x-goog-api-client` in the header? It's actually valid in the HTTP header spec to duplicate header keys.